sources/api/schnauzer/src/helpers.rs

```diff
@@ -106,6 +106,9 @@ const KUBE_RESERVE_ADDITIONAL: f32 = 2.5;
 const IPV4_LOCALHOST: IpAddr = IpAddr::V4(Ipv4Addr::new(127, 0, 0, 1));
 const IPV6_LOCALHOST: IpAddr = IpAddr::V6(Ipv6Addr::new(0, 0, 0, 0, 0, 0, 0, 1));
 
+const IMAGE_GC_HIGH_THRESHOLD_DEFAULT: &str = "85";
+const IMAGE_GC_LOW_THRESHOLD_DEFAULT: &str = "80";
```

Comment:
If we conditionally render the block in kubelet's config, we can avoid hardcoding the current default values, which might not stay the same over time.